chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+
+*What is our product? How will it saturate the market?*
+
+Our product will saturate the market mainly through the engagement with individuals/groups within the community and eventually towards our target market of news organizations and professional journalists.
+
+#### Key Features:
+*What makes it different from the competition?*
+            *see market share*
+
+*Price*
+Yellr will be priced on a tiered pricing system:
+
+| Feature                              | Bronze | Silver | Gold  |
+| -------                              | ------ | ------ | ----- |
+| Local Feed                           | x      | x      | x     |
+| Collections                          | x      | x      | x     |
+| Local Assignments <br> (25x25 miles) | x      | x      | x     |
+| Direct Messages to Users             |        | x      | x     |
+| Unlimited Post Storage               |        | x      | x     |
+| Infinite Assignment Area             |        |        | x     |
+| 24/7 phone support                   |        |        | x     |
+
+## Promotion
+
+Yellr is currently partnered with WXXI, an NPR affiliate which has done the majority of the promotion so far for the app. We are also reaching out to local journalism programs at RIT to test the deployment of the app as a hyper-local reporting methodology. Once these programs are finished we will review and perform a post-mortem. The post-mortem will focus on the reports of journalism students who used the app to gather sourcing and media content from communities.
+
+## Positioning/Place
+
+*Where will our app be sold?*
+
+Our app will be accessible from three main outlets:
+    - Google Play store
+    - iOS app store 
+    - Website
+
+##Financial Data: Assumptions and Forecasts 
+
+Yellr has sourced funding from various entities, including the Knight Foundation. Financial forecasts are determined by our strategies to generate revenue. We will generate revenue through:
+
+*Server Hosting for Organizations*
+    - Customization
+    - Featurization
+    - Tech Support
+    - Possible In-App Advertising
+
+Willingness to pay by news organizations and journalists is imperative to generating revenue for yellr. To leverage news organizations we must increase the number of downloads to increase the value of the product. We will be able to determine how to price our Bronze, Silver, and Gold packages for journalists and news organizations based on the number of downloads. The number of downloads will determine value for news organizations and journalists. We will use the number of downloads to determine the value of the yellr to journalists and news organizations. The forecasted downloads per quarter were conducted based on small percentages of the Rochester population of roughly 210,000 in 2013. We believe this product will be downloaded by 0.5% of Rochester’s population in the first quarter, 1.5% by quarter 2, 3% by Quarter 3, and 7% by the end of our first year for a total of 14,700 downloads. 
```

Comment:
Number of downloads isn't very likely to make organizations willing to pay. They are likely to care much more about activity and number of posts/comments/etc since those are more indicative of users actually using the app, not just placing an icon on their homescreen.